<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectDialog</class>
 <widget class="QDialog" name="ProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Screen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	border: 0;
	background: #444;
	padding: 8px;
}

QPushButton:hover {
	background: #555;
}

QListWidget {
	background: #404040;
	color: #DEDEDE;
}

QListWidget::item {
	padding: 6px;
}

QListView::item:selected {
    border: 1px solid #6a6ea9;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <property name="bottomMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="logo">
       <property name="styleSheet">
        <string notr="true">font-size: 32px;</string>
       </property>
       <property name="text">
        <string>JahshakaVR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newProject">
       <property name="text">
        <string>New Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openProject">
       <property name="text">
        <string>Open Project</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="head">
     <property name="text">
      <string>Recently Opened</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
